omniscidb/Utils/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 set(utils_source_files
     ChunkIter.cpp
-    CommandLineOptions.cpp
```

Comment:
I used this file to move boost program options init code into the config builder during a move of globals into the config structure. Didn't remove the file completely because we didn't move all globals into the config structure. But it stays in the git history anyway, so I don't mind removing it.